src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/Database/DatabaseHandler.cs

```diff
@@ -164,14 +163,23 @@ public override Task<InitializeViewResult> InitializeObjectView(InitializeViewRe
                                     EncryptionEnabled = smoDatabase.EncryptionEnabled
                                 };
 
+                                if (!isAzureDB) {
```

Comment:
This doesn't seem related to the description, is this just an additional fix or something?

Reply:
Oh, forgot to note that. Yes, it's an additional fix. We don't have the properties dialog enabled for Azure yet, so some of these properties haven't been excluded yet. An error gets thrown when trying to retrieve these properties against an Azure SQL DB.

Reply:
Delete _is_ enabled for Azure though, which is why I was hitting it.

Reply:
Note: We had a version of delete before for on-prem and azure DBs, but now that it uses a dialog it goes through this initialize view step.

Reply:
Already checked with @ssreerama and he aggreed.